src/Meta/Blueprint.php

```diff
@@ -260,17 +260,16 @@ public function references(self $table)
      */
     public function isUniqueKey(Fluent $constraint)
     {
-        $is = false;
-
         foreach ($this->unique as $index) {
-            foreach ($index->columns as $column) {
-                $is &= in_array($column, $constraint->columns);
-            }
-            if ($is) {
-                return true;
+
+            // We only need to consider cases, when UNIQUE KEY is presented by only ONE column
+            if (count($index->columns) === 1 && isset($index->columns[0])) {
+                if (in_array($index->columns[0], $constraint->columns)) {
```

Comment:
This code only works when the constraint is applied to a single column. However, you can add a UNIQUE KEY constraint that applies to more than one column. I think this is called "compound unique key" or "multiple-column index". How would you solve that?

Reply:
Well, I left a comment about that, though it may be not so clear. I thought through the problem and decided that in case of one-one relation, we never deal with compound unique keys. 

Let's mention example from the issue (Human-Brain). Can you imagine situation when Brain has compound key made of human_id and (?). Which key can be there instead of the question sign? 

Oh, wait... It can be skull_id... :) 

Give me a time...

Reply:
Well, even that example with skull_id is somewhat very unrealistic. I propose to merge this as is, my arguments are: 
1) this fixed my problem
2) let's be iterative, small steps are better than no progress
3) if someone needs that strange one-one relation with compound key, then let him think how to solve this too :) 
Or if you have suggestions, lets work this together, though I feel that it would be almost or absolutely useless work. 

What do you think?

Reply:
I'll merge this for now. However, I might change it before I publish the next release. Thanks for contributing. 👍

Reply:
Yes, would be cool if you mention me, i am curious what can be done here. Thank for merge! And waiting for release :)